### ISSUE TITLE
[rfcs] `no-space-importer` to trim importer

### ISSUE THREAD

**Issue by HomyeeKing** [import-js/eslint-plugin-import#2422]
I want to implement a new rule which to autofix the importer which has space 

```js
// case 1
const foo = require(' bar')
// case 2
import foo from '  bar'


// auto remove the space from the importer
```

but I wonder if it's proper to implement here, since I also want to handle the `require` scene and work in `js\ts`(they use the different ast)

so what's the correct way?  how about
- es import in this repo
- ts import in [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)
- require in [eslint](https://github.com/eslint/eslint)

**Comment by ljharb** [comment]
Spaces are allowed; trimming them just isn’t something you can do automatically.

The no-unresolved and no-extraneous-dependencies rules should be catching these already. Are they not?

**Comment by HomyeeKing** [comment]
sorry, I thought it not allowed having spaces in importer
